### PR TITLE
Add refplane kwarg + documentation to jpl horizons vectors_async method

### PR DESCRIPTION
**Pull request by mfixstsci**
Added `refplane` as a keyword arg that is passed to the request payload in the `vectors_async` method. I repurposed the code + documentation from the `elements` method.

### CODE REVIEW

File: astroquery/jplhorizons/core.py [hunk 1]
```diff
@@ -792,7 +792,7 @@ def elements_async(self, get_query_payload=False,
 
         return response
 
-    def vectors_async(self, get_query_payload=False,
+    def vectors_async(self, get_query_payload=False, refplane='ecliptic',
```
Comment (bsipocz) [inline astroquery/jplhorizons/core.py]: please add the new kwarg to the end of the signature so to avoid breaking the current API

**Comment by bsipocz** [comment]
cc @mommermi

**Comment by mommermi** [comment]
looks like a useful addition to me, so this can be merged after @bsipocz's comment has been addressed

**Comment by keflavich** [comment]
This looks like it's ready to go, though we'd ideally like a little better offline test coverage.

We also need a changelog entry.

**Comment by bsipocz** [comment]
> This looks like it's ready to go

Please don't merge it this state, it still needs a rebase as many unrelated commits are picked up.

**Comment by mfixstsci** [comment]
Hey @bsipocz , sorry I've let this go stale. I did a not so smart thing and merged the commits with master instead of rebasing... If I go ahead and rebase with the current master branch and push the changes up, would you be able to merge then?

**Comment by bsipocz** [comment]
@mfixstsci - Yes, I suppose this was ready to go in assuming tests pass and there is a changelog entry, too.

**Comment by mfixstsci** [comment]
@bsipocz- okay. I just went ahead and pulled down some changes and rebased but it looks like there was a change to `astropy_helpers`. The only difference is the `Subproject commit` hash changed. This shouldn't cause any problems when I push, should it? Just want to double check before pushing the changes up.

**Comment by bsipocz** [comment]
There should not be a change to the helpers. If you run  the tests locally, or run`git submodule update -- astropy_helpers` that change should go away.

**Comment by bsipocz** [comment]
@mfixstsci - rebase went wrong again. Let me know when you prefer me to step in and do it.

**Comment by mfixstsci** [comment]
@bsipocz- if you don't mind that would be great. I think I've wound this up into a mess that I can't get myself out of :(

**Comment by pllim** [comment]
@mfixstsci , I can help if you need help with this on-site. FYI.

**Comment by mfixstsci** [comment]
@lim - that would be great that way I can see how to properly rebase/see what I was doing wrong. Thanks @bsipocz I will clean this up push the changes on Monday!

**Comment by mfixstsci** [comment]
@bsipocz - @pllim just stopped by and we were able to clean everything up. Hopefully we should be ready to go now 👍

**Comment by bsipocz** [comment]
@mfixstsci - Thank you!

**Pull request merged**